.github/workflows/software-environments.yml

```diff
@@ -0,0 +1,42 @@
+name: Create software environments
+
+on:
+  push:
+    branches: main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 4
```

Comment:
Curious why this is needed

Reply:
I copied it over from https://github.com/coiled/software-environments/blob/main/.github/workflows/ci-update-envs.yml Should we remove those 4 lines? or just the strategy part?

Reply:
Just the `max-parallel: 4` part. I don't think we need it here. I see you removed that line -- thanks